src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/jpa/ResidentialLocation.kt

```diff
@@ -34,6 +34,7 @@ open class ResidentialLocation(
   childLocations: MutableList<Location>,
   whenCreated: LocalDateTime,
   createdBy: String,
+  usedFor: MutableSet<CellUsedFor>? = null,
```

Comment:
Not sure what this is for?

Reply:
I thought it was request but i will delete